website/static/js/home-page/donateBannerPlugin.js

```diff
@@ -0,0 +1,94 @@
+var m = require('mithril');
+var $osf = require('js/osfHelpers');
+
+// CSS
+require('css/donate-banner.css');
+require('css/meetings-and-conferences.css');
+
+var currentDate = new Date();
+
+var bannerOptions = [
+    {
+        startDate: new Date('August 7, 2017'),
```

Comment:
Drive-by: Better to construct a date with numbers rather than relying on parsing, which might break in certain browsers.

```js
new Date(2017, 8, 7)
```